multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -583,13 +587,18 @@ def _hyperparameter_tune(self, hyperparameter_tune_kwargs, resources, **_fit_arg
         metric = "val_" + _fit_args.get("validation_metric_name")
         mode = _fit_args.get("minmax_mode")
         save_path = _fit_args.get("save_path")
-        time_budget_s = _fit_args.get("max_time")
```

Comment:
Are we sure we want to remove this?

Reply:
Oh yes. This was when I tried to play around with the hanging issue. I'll add it back